package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.29.0",
   "description": "The easiest way to get started with Electron",
   "repository": "https://github.com/electron/fiddle",
-  "main": "./dist/src/main/main",
+  "main": "./dist/src/main/main.js",
```

Comment:
Small nit, but do we need this .js addition?

Reply:
We do. Couldn't tell you why, but `yarn make`/`yarn package` no longer worked with `beta64` without it. 🙂 So that was the only change that needed to be made to work with `beta64`, as far as I could tell.